Tests.csproj

```diff
@@ -2,8 +2,9 @@
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
this file should be in our git ignore, let's delete it from this PR before merging

Reply:
same with AssemblyReference.cache